homeassistant/components/google/calendar.py

```diff
@@ -443,3 +459,4 @@ async def async_create_event(entity: GoogleCalendarEntity, call: ServiceCall) ->
         )
     except ApiException as err:
         raise HomeAssistantError(str(err)) from err
+    await entity.async_update_ha_state(force_refresh=True)
```

Comment:
There's no update method that does anything. We can remove `True` from `async_add_entities` too.

```suggestion
    entity.async_write_ha_state()
```

Reply:
We can apply this suggestion too.